src/main/docker/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.3'
+services:
+  app:
+    image: 'docker-spring-boot-postgres:latest'
+    build:
+      context: .
+    container_name: app
+    env_file:
+      - .env
+    environment:
+      - SPRING_DATASOURCE_URL=jdbc:postgresql://db:${POSTGRES_PORT}/${POSTGRES_DB}
+      - SPRING_DATASOURCE_USERNAME=${POSTGRES_USERNAME}
+      - SPRING_DATASOURCE_PASSWORD=${POSTGRES_PASSWORD}
```

Comment:
You don't need those. `POSTGRES_USERNAME`, `POSTGRES_PASSWORD` and others should be in the `.env` file already.

```suggestion
```